example/src/main/java/io/netty/example/http/upload/HttpUploadServerHandler.java

```diff
@@ -224,6 +232,24 @@ private void readHttpDataChunkByChunk() {
                     }
                 }
             }
+            // Check partial decoding for a FileUpload
+            InterfaceHttpData data = decoder.currentPartialHttpData();
+            if (data != null) {
+                if (partialContent == null) {
+                    partialContent = (HttpData) data;
+                    if (partialContent instanceof FileUpload) {
+                        System.out.print("Start FileUpload: " + ((FileUpload) partialContent).getFilename() + " ");
```

Comment:
It is mainly an example on how to manage on server side the progression of the upload. Using the standard way of the example (building the answer and giving back to the client) is not clean from my perspective since it shall be print "during" upload, and not at the end (when the client get the final answer).
So I propose here a simple print from server side, as a log to show how to use the new capability. But if you have any better idea, for sure I will borrow it ;-)